src/ResourceManager/SignalR/Commands.SignalR/AzureRM.SignalR.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.1.0'
+ModuleVersion = '0.1.1'
 
 # Supported PSEditions
 # CompatiblePSEditions = @()
```

Comment:
@sergey-shandar update the following items in this file:
- Update the version of `AzureRM.Profile` to `5.0.0`
- Update the `ReleaseNotes` at the bottom with the new change log information